pages/index.tsx

```diff
@@ -75,6 +101,7 @@ export default function Home({
           href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,400;0,700;1,400;1,700&display=swap"
           rel="stylesheet"
         />
+        {enableGoogleAnalytics === 'enabled' && googleAnalytics}
```

Comment:
I think I might be being dense here. Doesn't this expression evaluate to `true`/`false`?

Reply:
It's a JSX feature, react conditional rendering: https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator

Reply:
😖

Reply:
oh wow Javascript doesn't do this natively? TIL